package.json

```diff
@@ -23,6 +23,7 @@
     "debug": "^4.3.1",
     "puppeteer": "^10.2.0",
     "ts-node": "^9.1.1",
+    "tslib": "^2.3.1",
```

Comment:
Is it needed?

Reply:
The tslib is required in my local build (windows 11).

Reply:
Here are the error messages when I ran `npm run server` from main branch:
src/bidiMapper/mapper.ts → src/.build/bidiMapper/mapper.js...
[!] (plugin typescript) Error: @rollup/plugin-typescript: Could not find module 'tslib', which is required by this plugin. Is it installed?
Error: @rollup/plugin-typescript: Could not find module 'tslib', which is required by this plugin. Is it installed?
    at error (C:\dev\chromium-bidi\node_modules\rollup\dist\shared\rollup.js:158:30)
    at throwPluginError (C:\dev\chromium-bidi\node_modules\rollup\dist\shared\rollup.js:21769:12)
    at Object.error (C:\dev\chromium-bidi\node_modules\rollup\dist\shared\rollup.js:22472:20)
    at preflight (C:\dev\chromium-bidi\node_modules\@rollup\plugin-typescript\dist\index.js:495:17)
    at Object.buildStart (C:\dev\chromium-bidi\node_modules\@rollup\plugin-typescript\dist\index.js:736:13)
    at C:\dev\chromium-bidi\node_modules\rollup\dist\shared\rollup.js:22681:25
    at async Promise.all (index 0)
    at rollupInternal (C:\dev\chromium-bidi\node_modules\rollup\dist\shared\rollup.js:23397:9)
    at build (C:\dev\chromium-bidi\node_modules\rollup\dist\bin\rollup:1547:20)
    at runRollup (C:\dev\chromium-bidi\node_modules\rollup\dist\bin\rollup:1731:21)